backend/gateway/mux/mux.go

```diff
@@ -62,14 +69,53 @@ func (a *assetHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	// Serve!
 	if f, err := a.fileSystem.Open(r.URL.Path); err != nil {
+		// If not a known static asset and an asset provider is configured, try streaming from the configured provider.
+		if a.assetCfg != nil {
+			if a.assetCfg.Provider != nil && strings.HasPrefix(r.URL.Path, "/static") {
```

Comment:
though collision is unlikely this is slightly more correct
```suggestion
			if a.assetCfg.Provider != nil && strings.HasPrefix(r.URL.Path, "/static/") {
```